framework/auth/cas.py

```diff
@@ -208,15 +213,23 @@ def get_logout_url(*args, **kwargs):
     """
     return get_client().get_logout_url(*args, **kwargs)
 
+
 def get_profile_url():
     """Convenience function for getting a profile URL for a user.
     """
     return get_client().get_profile_url()
 
+
```

Comment:
Agreed. However, In code that have both 1 and 2 blank lines, do we prefer to make the style consistent or just leave them the way it is.